en/reads-and-writes.html

```diff
@@ -31,7 +31,7 @@
   and multiple container clusters - see <a href="indexing.html">indexing</a> for details.
 </li>
 <li>
-  Writes in Vespa are <em>consistent</em>, but Vespa will prioritize availability over consistency when
+  Writes in Vespa are <em>eventually consistent</em>, but Vespa will prioritize availability over consistency when
```

Comment:
How do you think it's "eventually"? Usually this means "after a write I may get the old or the new value, but eventually it will converge to the new".

Reply:
> Maybe it would be better to rephrase to state that in a stable system, writes are consistent, but otherwise we will prioritize availability?

Yes - I think that is better. That is our design goal and people usually get it when explained in this way.

Reply:
@bratseth I've changed the wording here, PTAL